DEVELOPMENT.md

```diff
@@ -0,0 +1,18 @@
+Development Setup
+===============
+This document provides recommendations for setting up a develpment environment
+to facilitate reproducible research.
+# Github Rulesets
+Github rulesets define rules for interacting with a repository.  They are
+intended to improve code quality through better code review and merging
+practices.
+
+A Github ruleset can be setup by importing a ruleset JSON file.
+To setup a ruleset follow these steps:
+* Navigate to the Github repository.
```

Comment:
I changed the instructions to download the settings file from the repository on Github and then upload it to settings.  This way, it will work regardless of where they are.  Thanks for the feedback!!